android/src/main/java/com/thebluealliance/androidclient/gcm/notifications/BaseNotification.java

```diff
@@ -25,12 +26,16 @@
 import com.thebluealliance.androidclient.listitems.ListElement;
 import com.thebluealliance.androidclient.models.StoredNotification;
 import com.thebluealliance.androidclient.receivers.NotificationChangedReceiver;
+import com.thebluealliance.androidclient.viewmodels.GenericNotificationViewModel;
+import com.thebluealliance.androidclient.viewmodels.ViewModelRenderer;
 
 import java.text.DateFormat;
 import java.util.Calendar;
 import java.util.Date;
 
-public abstract class BaseNotification extends ListElement {
+import io.nlopez.smartadapters.views.BindableLayout;
+
+public abstract class BaseNotification<VIEWMODEL> extends ListElement implements ViewModelRenderer<VIEWMODEL, Void> {
```

Comment:
The only reason I had for using generics for ViewModelAdapter was for type safety. If for some reason we had to know the exact return type of `renderToViewModel()` for a certain model, we could check that instead of having to use a generic `ParentModelView`, which at this point would be a useless empty class.